chargebee/src/main/java/com/chargebee/android/billingservice/BillingClientManager.kt

```diff
@@ -333,9 +431,21 @@ class BillingClientManager(context: Context) : PurchasesUpdatedListener {
 
             else -> {
                 if (purchaseProductParams.product.type == ProductType.SUBS)
-                    purchaseCallBack?.onError(
-                        throwCBException(billingResult)
-                    )
+                    if (this.changeProductParams.oldProductId.isNotEmpty() && billingResult.responseCode == ERROR) {
```

Comment:
Where do we reset the `changeProductParams` variable?

Reply:
Done